pkg/services/sqlstore/sqlstore.go

```diff
@@ -333,7 +333,7 @@ func (ss *SQLStore) readConfig() {
 	ss.dbCfg.CacheMode = sec.Key("cache_mode").MustString("private")
 }
 
-// Interface of arguments for testing db
+// ITestDB Interface of arguments for testing db
```

Comment:
[Comments documenting declarations should be full sentences](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences)
```suggestion
// ITestDB is an interface of arguments for testing DB
```